inst/ms/aggreCAT.qmd

```diff
@@ -94,21 +126,21 @@ and aims to build automated tools that can rapidly and reliably assign
 "Confidence Scores" to research claims from empirical studies in the
 Social and Behavioural Sciences (SBS). Confidence Scores are
 quantitative measures of the likely reproducibility or replicability of
-a research claim or result, and may be used by consumers of Social and
-Behavioural Sciences research as a proxy measure for their credibility
-in the absence of replication effort.
+a research claim or result, and may be used by consumers of scientific
+research as a proxy measure for their credibility in the absence of
+replication effort.
 
 Replications are time-consuming and costly [@Isager2020], and studies
 have shown that replication outcomes can be reliably elicited from
 researchers [@Gordon2020]. Consequently, the DARPA SCORE program
 generates Confidence Scores using expert elicitation based on two very
 different strategies -- prediction markets [@Gordon2020] and the IDEA
 protocol [@hemming2017], the latter of which is used by the repliCATS
-project [@Fraser:2021]. **X** of these research claims were randomly
-selected for direct replication, against which the elicited Confidence
-Scores are 'ground-truthed'. These findings will aid the development of
-artificial intelligence tools that can automatically assign Confidence
-Scores.
+project @Fraser:2021. **X** of these research claims were randomly
```

Comment:
Number has since changed, and the full set not completed. We (me and @Doi90 ) decided in our meeting today that we would go with "A proportion of ..."